src/annotator/test/test_msomajo.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import annotator.msomajo as msm
+
+
+@pytest.fixture
+def read_test_de():
+    with open("test/test_files/example_de_somajo.vrt", "r") as f:
+        data = f.read()
+    return data
+
+
+@pytest.fixture
+def read_test_en():
+    with open("test/test_files/example_en_somajo.vrt", "r") as f:
+        data = f.read()
+    return data
+
+
+def test_tokenize(read_test_de, read_test_en):
+
+    text_de = "Das Feld der Computerchemie erfreut sich seit einiger Zeit einer stetig zunehmenden Aufmerksamkeit. Hierbei ist es für die Berechnung von chemisch relevanten Systemen von Interesse, möglichst große Systeme (Moleküle) durch effiziente Nutzung von Rechen- und Speicherressourcen zugänglich zu machen. Die Herausforderung besteht darin, dass mit der zunehmenden Anzahl von Atomen in größeren Systemen auch die Anzahl der internen Freiheitsgrade zunimmt."
```

Comment:
The tests are saved in `test_files/example_en.txt` and `test_files/example_de.txt`, for keeping it general should read from there.